src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -198,13 +230,55 @@ const SendFunds = ({ initialValues, onClose, onNext, recipientAddress, selectedT
                     />
                   </Col>
                 </Row>
+                {tokenSpendingLimit && (
+                  <Row margin="sm">
+                    <Col between="lg" style={{ flexDirection: 'column' }}>
+                      <Text size="lg">Send as</Text>
+                      <Field name="txType" initialValue="multiSig">
+                        {({ input: { name, value } }) => (
+                          <SpendingLimitRadioButtons
+                            name={name}
+                            value={value || 'multiSig'}
+                            onRadioChange={mutators.setTxType}
+                            options={[
+                              { label: 'Multisig Transaction', value: 'multiSig' },
+                              {
+                                label: `Spending Limit Transaction (${fromTokenUnit(
+                                  new BigNumber(tokenSpendingLimit.amount).minus(tokenSpendingLimit.spent).toString(),
+                                  selectedTokenRecord.decimals,
+                                )} ${selectedTokenRecord.symbol})`,
+                                value: 'spendingLimit',
+                              },
+                            ]}
+                          />
+                        )}
+                      </Field>
+                    </Col>
+                  </Row>
+                )}
                 <Row margin="xs">
                   <Col between="lg">
                     <Paragraph color="disabled" noMargin size="md" style={{ letterSpacing: '-0.5px' }}>
                       Amount
                     </Paragraph>
                     <ButtonLink
-                      onClick={() => mutators.setMax(selectedTokenRecord.balance)}
+                      onClick={() =>
```

Comment:
The component is getting really big, would be good to split it

Reply:
https://github.com/gnosis/safe-react/issues/1317